bundles/org.openhab.ui/web/src/assets/definitions/widgets/standard/cards.js

```diff
@@ -24,8 +24,7 @@ export const OhLabelCardDefinition = () => new WidgetDefinition('oh-label-card',
     pt('background', 'Background style', 'Background style (in CSS "background" attribute format)'),
     pt('fontSize', 'Font Size', 'Font size (e.g. "34px")'),
     pt('fontWeight', 'Font Weight', 'Font weight (e.g. "normal" or "bold")'),
-    pt('icon', 'Icon', 'Use <code>oh:iconName</code> (<a class="external text-color-blue" target="_blank" href="https://www.openhab.org/link/icons">openHAB icon</a>) or <code>f7:iconName</code> (<a class="external text-color-blue" target="_blank" href="https://framework7.io/icons/">Framework7 icon</a>)'),
-    pt('iconColor', 'Icon Color', 'Not applicable to openHAB icons').a(),
+    pt('icon', 'Icon', 'Use <code>oh:iconName</code> (<a class="external text-color-blue" target="_blank" href="https://www.openhab.org/link/icons">openHAB icon</a>), <code>f7:iconName</code> (<a class="external text-color-blue" target="_blank" href="https://framework7.io/icons/">Framework7 icon</a>) or <code>material:iconName</code> (<a class="external text-color-blue" target="_blank" href="https://jossef.github.io/material-design-icons-iconfont/">Material icon</a>)'),    pt('iconColor', 'Icon Color', 'Not applicable to openHAB icons').a(),
```

Comment:
Please add the line break back.

Reply:
Huh, don‘t even know how it disappeared. Will add it back of course.